src/Symfony/Component/DependencyInjection/Dumper/PhpDumper.php

```diff
@@ -1137,6 +1199,39 @@ private function addAliases()
         return $code."        );\n";
     }
 
+    private function addInlineRequires()
+    {
+        if (!$this->hotPathTag || !$this->inlineRequires) {
```

Comment:
Why have two ways to disable this? Can you just make hotPathTag required?

Reply:
Because these are two independent settings (that play well together.)